dm/syncer/err-operator/operator.go

```diff
@@ -110,17 +135,17 @@ func (h *Holder) GetEvent(startLocation binlog.Location) (*replication.BinlogEve
 	key := startLocation.Position.String()
 	operator, ok := h.operators[key]
 	if !ok {
-		return nil, terror.ErrSyncerReplaceEventNotExist.Generate(startLocation)
+		return nil, terror.ErrSyncerEventNotExist.Generate(startLocation)
 	}
 
 	if len(operator.events) <= startLocation.Suffix {
-		return nil, terror.ErrSyncerReplaceEvent.Generatef("replace events out of range, index: %d, total: %d", startLocation.Suffix, len(operator.events))
+		return nil, terror.ErrSyncerEvent.Generatef("replace or inject events out of range, index: %d, total: %d", startLocation.Suffix, len(operator.events))
 	}
 
 	e := operator.events[startLocation.Suffix]
 	buf := new(bytes.Buffer)
 	e.Dump(buf)
-	h.logger.Info("get replace event", zap.Stringer("event", buf))
+	h.logger.Info("get replace or inject event", zap.Stringer("event", buf))
```

Comment:
we should update the comment of this func, add sth about `inject`

Reply:
yes, it will be clearer.